src/services/cli-http-client.js

```diff
@@ -1,15 +1,21 @@
+var http_ = require('http');
+var https = require('https');
 const os = require('os');
 const pkg = require('../../package.json');
+const qs = require('qs');
 const { TwilioCliError } = require('../services/error');
 const { NETWORK_ERROR } = require('../services/messaging/help-messages');
 
-const NETWORK_ERROR_CODES = new Set(['ETIMEDOUT', 'ESOCKETTIMEDOUT']);
+const NETWORK_ERROR_CODES = new Set(['ETIMEDOUT', 'ESOCKETTIMEDOUT', 'ECONNABORTED']);
 
 class CliRequestClient {
   constructor(commandName, logger, http) {
     this.commandName = commandName;
     this.logger = logger;
-    this.http = require('util').promisify(http || require('request'));
+    this.http = http || require('axios');
+    if (!http) {
```

Comment:
Don't think this check is needed. There should always be a client.

Reply:
I was trying to check if the user provided a custom http client, because I didn't necessarily want to overwrite their content-type defaults. Just want to set this if we're using the default axios client. Should this override user defaults?

Reply:
Maybe just remove it since the upstream client `TwilioApiClient` sets it.

Reply:
Ah. gotcha

Reply:
@childish-sambino so removing this actually breaks a unit test which dummies the http client. I think this should stay, even if only for testability.

Reply:
Maybe move the content-type header (not default header) logic to this class instead?

Reply:
Well, as it turns out it already does that. I'll take this bit out then